doc_template/examples_root/examples/nb/visual_behavior_ophys_dataset_manifest.ipynb

```diff
@@ -59,172 +59,81 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    fig = plt.hist(cell_per_exp, bins=50)
</pre><p>Could we add the figure axis labels here?</p><p>Something like</p><p>plt.xlabel("Cell count")</p><p>plt.ylabel("Number of experiments")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/AllenSDK/pull/2620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='169'/>

Reply:
Thanks, I added that